axelrod/tests/unit/test_match_generator.py

```diff
@@ -44,7 +44,7 @@ def test_len(self):
         with self.assertRaises(NotImplementedError):
             len(tt)
 
-    def test_build_match_params(self):
+    def test_build__params(self):
```

Comment:
I see you fixed this but I would just fix it (and the others) back to what it was (just read the diff here):

```
test_build_match_params
```